spec.html

```diff
@@ -28420,29 +28274,19 @@ <h1>Math.random ( )</h1>
       <emu-clause id="sec-math.round">
         <h1>Math.round ( _x_ )</h1>
         <p>Returns the Number value that is closest to _x_ and is an integer. If two integers are equally close to _x_, then the result is the Number value that is closer to *+&infin;*. If _x_ is already an integer, the result is _x_.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *-&infin;*.
-          </li>
-          <li>
-            If _x_ is greater than 0 but less than 0.5, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is less than 0 but greater than or equal to -0.5, the result is *-0*.
-          </li>
-        </ul>
+        <emu-alg>
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is *NaN*, _n_ is *-0*, _n_ is *+0*, _n_ is *+&infin;*, or _n_ is *-&infin;*, return _n_.
+          1. If _n_ &gt; 0 and _n_ &lt; 0.5, return *+0*.
+          1. If _n_ &lt; 0 and _n_ &lt;= -0.5, return *-0*.
+          1. Let _r_ be _n_ modulo 1.
+          1. Let _v_ be _n_ - _r_.
+          1. If _r_ is 0, return _v_.
+          1. If _r_ &gt;= -0.5 and _r_ &lt; 0, return _v_.
+          1. If _r_ &lt; 0.5 and _r_ &gt; 0, return _v_.
+          1. If _r_ &lt; -0.5, return _v_ - 1.
+          1. Return _v_ + 1.
```

Comment:
I think we can say this more simply and equally clearly with prose:

```suggestion
          1. If _n_ is an integral Number, return _n_.
          1. Return the integral Number closest to _n_, preferring the Number closer to *+&infin;* in the case of a tie.
```

Reply:
That's fine. It was unclear to me to how close these should be to implementation algorithms so I probably went too far on this one.